### ISSUE TITLE
Quest: Broodling Essence - Burning Steppes

### ISSUE THREAD

**Issue by Katherwind** [WoWManiaUK/Redemption#1492]
Bugged Quest Link:   http://www.wowhead.com/quest=4726/broodling-essence

The quest works as follows:  fire the Draco-Incarcinatrix 900 at the whelplings ,then kill them.  A crystal will spawn which you loot for the essence.  The crystal containing the essence is not spawning after the whelp is killed.
 
I have tested this on quite a few of the whelps and have noticed that if there is a red patch on the ground by their spawn point, the essence crystal will not spawn. I was able to complete the quest by searching the zone for mobs without the red patch on the ground by their spawn point.  However, I think I got the last  remaining "untapped" whelps in the zone.  Anyone trying to do the quest after me will not be able to complete the quest.

**Comment by tvshadowz** [comment]
Hey Katherwind!

In future please make sure you link all items used in the quest aswell as it helps us testers a ton at times. But I will test this asap after I'm doing testing the current bug. 

Thanks for the report!

**Comment by tvshadowz** [comment]
Hey Katherwind! 

I found that the crystal did not spawn from time to time after doing everything necessary. And as you explain those who's spawn points are closer to red circles on the ground (see screenshot) are less likely to drop the crystals. 

![test drag](https://user-images.githubusercontent.com/30256554/29426047-496f3820-837d-11e7-8697-540ad9e73b70.png)

I was able to complete the quest however with some issue. 

Thank you for the report!
